backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/ClusterBuilder.java

```diff
@@ -186,8 +184,18 @@ public Cluster build() {
         }
     }
 
+    private QueryOptions queryOptions() {
+        QueryOptions queryOptions = new QueryOptions()
+                .setConsistencyLevel(ConsistencyLevel.QUORUM);
+
+        getRefreshSchemaIntervalMillis().ifPresent(refreshSchemaIntervalMillis ->
+                    queryOptions.setRefreshSchemaIntervalMillis(refreshSchemaIntervalMillis));
+
+        return queryOptions;
+    }
+
     private Optional<Integer> getRefreshSchemaIntervalMillis() {
-        return forTest ? Optional.of(0) : refreshSchemaIntervalMillis;
```

Comment:
why did you removed that ?

Reply:
This feature was buggy & is not used (the _ClusterBuilder.refreshSchemaIntervalMillis_ method is never called).
Do you want that I remove it, I hesitate on that point?

I think it was used in order to optimize the tests previously.

Reply:
I would start the PR with a commit containing only that part and a comment that explains it's not in use

Reply:
so forTest seems to not be used anymore, can you remove it?

Reply:
That was my question...